src/commands/deploy.rs

```diff
@@ -1,13 +1,20 @@
-use anyhow::{Context, Result};
+use anyhow::{anyhow, Context, Result};
 use bindle::Id;
 use clap::Parser;
+use data_encoding::HEXLOWER;
 use hippo::{Client, ConnectionInfo};
 use hippo_openapi::models::ChannelRevisionSelectionStrategy;
-use spin_loader::local::config::RawAppManifestAnyVersion;
+use ring::digest::{Digest, SHA256};
```

Comment:
Can we switch to the `sha2` crate here? We don't currently have a direct dependency on `ring` but we do use `sha2` in a few places.

Reply:
Updated